lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app/src/main/java/io/lighty/applications/rcgnmi/app/Arguments.java

```diff
@@ -20,11 +22,35 @@
             + " (If absent, will look on classpath for it")
     private String loggerPath;
 
+    @Parameter(names = {"-t", "--timeout-in-seconds"}, validateWith = ApplicationTimeoutValidator.class,
+               description = "Application timeout in seconds. "
```

Comment:
Application start-up timeout in seconds. Timeout exception is thrown when application fails to start within the specified time. Default value is 60. (range: 15 - Integer.MAX_VALUE)